.github/scripts/windows/test_task.bat

```diff
@@ -54,7 +55,7 @@ set PDOTEST_DSN=odbc:%ODBC_TEST_DSN%
 
 rem setup Firebird related exts
 curl -sLo Firebird.zip https://github.com/FirebirdSQL/firebird/releases/download/v3.0.9/Firebird-3.0.9.33560-0_x64.zip
-7z x -oC:\Firebird Firebird.zip
+7z x -y -oC:\Firebird Firebird.zip
```

Comment:
this flag answers yes to overwriting files if run more than once. As a result the cleanup.bat and other such scripts provide improvement (can't remove or overwrite if firebird exists, which it never will in GitHub actions)